src/webpack/loaderUtils/configHelpers.js

```diff
@@ -20,8 +23,21 @@ const pageTypes = primaryNavigationItems => (primaryNavigationItems.reduce((acc,
  * Provides the namespace for the package in this order, mono repo package, node_modules package, provided package name.
  */
 const getNamespace = (directory, namespace) => {
-  const afterPackages = (/packages\/([^/]*)/.exec(directory) || {})[1];
-  const afterNodeModules = (/node_modules\/([^/]*)/.exec(directory) || {})[1];
+  // If this is a monorepo package, we need to pull the namespace from the package.json file to account for scoping.
+  const packageRoot = (/.*packages\/[^/]*/.exec(directory) || {})[0];
+  let afterPackages;
+  if (packageRoot) {
+    const packagePath = path.join(packageRoot, 'package.json');
+    // cache the package name to avoid opening the files all the time.
+    afterPackages = monoRepoPackageCache[packagePath];
+    if (!afterPackages && fs.existsSync(packagePath)) {
+      // eslint-disable-next-line global-require, import/no-dynamic-require
+      afterPackages = require(packagePath).name;
+      monoRepoPackageCache[packagePath] = afterPackages;
+    }
+  }
+  // Find the directory name directly after node modules... include scoped package if they exist.
+  const afterNodeModules = (/node_modules\/((@[^/]*\/)?[^/]*)/.exec(directory) || {})[1];
```

Comment:
i think i need to brush up on my regex, but should `*` be swapped with `+` to match 1 or more, rather than 0 or more?

Reply:
Technically I think you're right, but functionally they're the same since the capture group won't capture zero characters between slashes.